### DISCUSSION TITLE
CommandLineOptions

### DISCUSSION THREAD

**Discussion by iamq-dev** [d2phap/ImageGlass#969] [Q&A] [answered]
Are there command line options so that I could set up a hotkey to open ImageGlass with the clipboard data? Say via autohotkey. ( alternatively it can just open ImageGlass then Crtl+V,  but I wanted to check for options first)

Thanks!

**Comment by d2phap** [comment, marked as answer]
Hi @TheLordOfTheQ 
You can check the command line utilities here: https://imageglass.org/docs/command-line-utilities
